Holacracy-Constitution.md

```diff
@@ -12,7 +12,7 @@ This **_“Constitution”_** defines rules and processes for the governance and
 The Organization’s Partners will typically perform work for the Organization by acting in an explicitly defined Role. A **_“Role”_** is an organizational construct with a descriptive name and one or more of the following:
```

Comment:
Now that I read again this line, I see that my proposal is not valid. But anyway, maybe there is a danger of leaving the text as it is. Having Purpose, Accountability and Domains presented like that can let people imagining that the three elements are required to create a role.

And as a side note: Is it possible to have role without purpose or accountability, only a Domain?